apps/frontend/static_src/js/main.js

```diff
@@ -1 +1,70 @@
 import { Dropdown } from 'bootstrap';
+
+const searchInput = document.querySelector('[data-search-input]')
+const searchIconButton = document.querySelector('[data-search-icon-button]')
+
+const onSearchInputChange = async (event) => {
+    const query = event.target.value
+    try {
+        const res = await fetch(window.location.origin + '/search_json/?query=' + query)
```

Comment:
Not sure this matters here but we want to be careful with how we handle input like this, always URL-encoding it as needed.

```suggestion
        const res = await fetch(`/search_json/?${(new URLSearchParams({ query })).toString()}`)
```

Reply:
We have language/version slug in every page. Therefore `/search_json/` would return `/en-latest/search_json` instead of `/search_json` which is what we need. For this reason I have used `window.location.origin`.
Apart from this I will add URL-encoding as you suggested above.